src/ui/pages/Overview/Overview.tsx

```diff
@@ -398,6 +408,24 @@ function OverviewComponent() {
     </CenteredFillViewportView>
   );
 
+  /**
+   * Creates href such that "chain" search-param is preserved between
+   * tabs, but clicking on current tab resets searchParams
```

Comment:
Right now clicking on selected tab scrolls you to the top. Is it ok to combine 2 actions here (scroll + filter reset)?

Reply:
We have `All networks` button for this quick filter reset